include/deal.II/base/mpi.h

```diff
@@ -1102,143 +1104,49 @@ namespace Utilities
            const unsigned int root_process = 0);
 
 
-    template <typename T>
-    struct is_mpi_type
-    {
-      static const bool value = false;
-    };
-
-
-
-    template <>
-    struct is_mpi_type<bool>
-    {
-      static const bool value = true;
-    };
-
-
 
-    template <>
-    struct is_mpi_type<char>
-    {
-      static const bool value = true;
-    };
-
-
-
-    template <>
-    struct is_mpi_type<signed char>
-    {
-      static const bool value = true;
-    };
+    template <bool...>
+    struct check
+    {};
 
 
 
-    template <>
-    struct is_mpi_type<short>
+    template <typename U, typename... T>
+    constexpr auto contains(std::tuple<T...>)
```

Comment:
There is only one possible  return type here that will not change with the implementation (and it has few characters).
```suggestion
    constexpr bool contains(std::tuple<T...>)
```

Reply:
Fixed in 74a317d